core/schema/container.graphqls

```diff
@@ -88,6 +88,18 @@ type Container {
   "Retrieves this container plus the given environment variable."
   withEnvVariable(name: String!, value: String!): Container!
 
+  "Retrieves the list of labels passed to container."
```

Comment:
Nit: for consistency
```suggestion
  "Retrieves the list of labels passed to this container."
```

Reply:
I checked on all the codebase, there is no mention of `to this` anywhere. I followed the consistency of envVariables